packages/styles/src/carousel.scss

```diff
@@ -152,8 +152,7 @@ $fd-carousel-button-content-offset: 0.5rem !default;
     @include fd-reset();
     @include fd-flex-center();
 
-    min-height: 2.75rem;
-    max-height: 3.5rem;
```

Comment:
I think this is not right. Now all page indicators, with and without buttons, have the same height of 2.75rem. 
<img width="479" alt="Screenshot 2023-11-07 at 12 02 34 PM" src="https://github.com/SAP/fundamental-styles/assets/39598672/beb18d94-6688-4598-a6b5-7477fd6d2b9a">

**Per specs, the height of the page indicator with buttons should be 3.5rem**
<img width="409" alt="Screenshot 2023-11-07 at 12 02 51 PM" src="https://github.com/SAP/fundamental-styles/assets/39598672/583846d3-fd17-4f21-9f86-42bb696e7ae7">